src/Service/ActivityPubManager.php

```diff
@@ -419,11 +421,19 @@ public function handleImages(array $attachment): ?Image
 
         if (\count($images)) {
             try {
-                if ($tempFile = $this->imageManager->download($images[0]['url'])) {
+                $imageObject = $images[0];
+                if (isset($imageObject['height'])) {
+                    foreach ($images as $i) {
```

Comment:
Please add code comment explaining why this is done. So in the future it's still understood why.

Reply:
I mean that part of the code is pretty obvious isn't it? I mean yeah the code for knowing whether to wrap it in an array or not makes sense to me

Reply:
now you do, but also over 6 years from now?